modules/services/emacs.nix

```
@@ -80,6 +80,16 @@ in {
     socketActivation = {
       enable = mkEnableOption "systemd socket activation for the Emacs service";
     };
+
+    defaultEditor = mkOption rec {
+      type = types.bool;
+      default = false;
+      example = !default;
+      description = ''
+        Whether to configure <command>emacsclient</command> as the default
+        editor using the <varname>EDITOR</varname> environment variable.
```

```suggestion
        editor using the <envar>EDITOR</envar> environment variable.
```
The more specific `envar` is preferred when referring to environment variables.